apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -984,9 +984,6 @@ void wm_auto_test(void *arg)
 
 		printf("[WT] Cycle finished [Round %d]\n", cnt);
 	}
-	printf("[WT] Exit WiFi Manager Stress Test..\n");
```

Comment:
Is this related with SVACE?

Reply:
Yes. 
WID:10623772 This statement in the source code might be unreachable during program execution.